src/components/wallet/views/claimableDisplay.tsx

```diff
@@ -0,0 +1,58 @@
+import { h } from '@stencil/core'
+import { has as loHas } from 'lodash-es'
+import WalletButton from './walletButton'
+import { Wallet } from '../wallet'
+
+interface ClaimableBalance {
+  id: string
+  asset: string
+  amount: string
+  sponsor?: string
+  claimants: any[]
+}
+
+export default function claimableDisplay(this: Wallet) {
+  if (this.account.availableBalances) {
```

Comment:
I'm not sure if this is the right code (is there a difference between `this.account.claimableBalances` and `claimableBalanceResp` used to populate `this.account.availableBalances`?), but you get the gist of the suggestion:
```
  if (this.account.claimableBalances.records.length !== 0) {
```